Xtimate

upayanmazumder opened the pull request.

g9aerospace commented:
Updates have been done mostly!

Hero is like this now:
![image](https://github.com/Eranodes/eranodes.com/assets/80208426/a664fc55-7a06-494c-abdc-219eaa1bf3e9)


Plans section:
![image](https://github.com/Eranodes/eranodes.com/assets/80208426/e6cd5a11-bdd6-4479-8ef9-4bfb9937308a)

Map and footer:
![image](https://github.com/Eranodes/eranodes.com/assets/80208426/f050f675-efa6-4533-a49e-43b586a90935)

Xtimate committed:
Installed vite for dev server

Xtimate committed:
updates

Xtimate committed:
Hid the scrollbar

Xtimate committed:
Changed the scrollbar

Xtimate committed:
Perfect scrollbar

Xtimate committed:
Installed Tailwind CSS

upayanmazumder committed:
updates

upayanmazumder committed:
added granular control

upayanmazumder committed:
🔴 space/space.tsx
🔴 space/space.module.css
🟢 testimonials.module.css
🛠️ testimonials.tsx -> Updated container class
🛠️ index.tsx -> Removed import for Space component, restructured the layout.

Xtimate committed:
Fixed footer

Xtimate committed:
Added tailwind css tags

upayanmazumder committed:
gg

upayanmazumder committed:
Merge branch 'xtimate' of https://github.com/Eranodes/eranodes.com into xtimate

upayanmazumder committed:
🛠️ nav.module.css -> Added text-align: left
🛠️ hero.module.css -> Modified padding to 15px 30px
🛠️ styles.css -> Adjusted position of ellipsis-aqua

upayanmazumder committed:
fixed here

upayanmazumder committed:
Merge branch 'main' into xtimate
